src/libraries/System.Runtime.InteropServices.NFloat.Internal/ref/System.Runtime.InteropServices.NFloat.Internal.csproj

```diff
@@ -17,7 +17,6 @@
     <DefineConstants>$(DefineConstants);FEATURE_NFLOAT</DefineConstants>
     <IsPackable>true</IsPackable>
     <ServicingVersion>1</ServicingVersion>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
I'd defer to @ericstj here.

This is a "new" package, not one that shipped in .NET 6 RTM and is the "special" package meant to support Xamarin/MAUI.

I'm uncertain if there is any special consideration due to that.

Reply:
Does Xamarin/MAUI expect us to give them a new build every time we service, or only if there are changes?

Reply:
I believe only if there are changes. But we should confirm with @rolfbjarne

Reply:
I don't think we need a new version unless something changes, and in any case I hardcoded the version we reference as 6.0.1, so it would also make our lives easier if the version didn't bump unnecessarily :)

Reply:
Thank you everyone. So the change as it is right now seems to be the correct one.